lightning/src/ln/blinded_payment_tests.rs

```diff
@@ -1416,6 +1435,60 @@ fn custom_tlvs_to_blinded_path() {
 	);
 }
 
+#[test]
+fn fails_receive_tlvs_authentication() {
```

Comment:
@valentinewallace I cribbed this off some of the other tests here. Let me know if something looks off or if it could be done in a simpler manner.